### PR TITLE
Added migration year to ouput

### PR DESCRIPTION
**Pull request by tylerstright**

### CODE REVIEW

File: R/get_PITobs.R [hunk 1]
```diff
@@ -36,6 +36,7 @@ get_PITobs = function(query_type = c('obs_site', 'release_site'),
   query_type <- match.arg(query_type)
 
   stopifnot(!is.null(start_date)|!is.null(end_date))
+  stopifnot(!is.na(as.Date(start_date, format ='%m/%d/%Y'))|!is.na(as.Date(end_date, format ='%m/%d/%Y')))
```
Comment (ryankinzer) [inline R/get_PITobs.R]: I wonder if we could check the format of the dates supplied but not explicitly change them to dates?  An actually date value isn't needed for the url params.

File: R/get_PITobs.R [hunk 2]
```diff
@@ -215,6 +215,11 @@ get_PITobs = function(query_type = c('obs_site', 'release_site'),
       between(lubridate::month(release_date),7,8) ~ 'Summer',
       lubridate::month(release_date) >= 9 ~ 'Fall',
       TRUE ~ 'Unknown'
+    ),
```
Comment (ryankinzer) [inline R/get_PITobs.R:218]: This was simple.

File: R/get_PITobs.R [hunk 3]
```diff
@@ -46,11 +46,11 @@ get_PITobs = function(query_type = c('obs_site', 'release_site'),
   rear_type <- match.arg(rear_type, several.ok = FALSE)
 
   # set up default start and end days
-  start_day <- substr(start_date,1,5)
-  syear <- substr(start_date, 7,10)
+  start_day <- gsub('-','/',substr(as.Date(start_date, format ='%m/%d/%Y'),6,10))
```
Comment (ryankinzer) [inline R/get_PITobs.R]: If we figure out a way to ensure the string format is followed for the date values I think this is more complex than it needs to be.

**Review by ryankinzer — CHANGES_REQUESTED** [review]
I'm trying out these comments.  How do they work?

File: R/get_PITobs.R (hunk 1 above)
Comment (tylerstright) [reply R/get_PITobs.R]: See next commit!

File: R/get_PITobs.R (hunk 2 above)
Comment (tylerstright) [reply R/get_PITobs.R:218]: Elegant.

**Review by tylerstright — COMMENTED** [review]
This should be better - although a very problematic person could input 99/99/9999 and have it work.

**Comment by ryankinzer** [comment]
grepl......who would have thought.

**Pull request merged**